octoprint_mrbeam/__init__.py

```diff
@@ -2318,8 +2318,11 @@ def take_undistorted_picture(self, is_initial_calibration):
             "New undistorted image is requested. is_initial_calibration: %s",
             is_initial_calibration,
         )
-        self.lid_handler._photo_creator.is_initial_calibration = is_initial_calibration
-        self.lid_handler._startStopCamera("initial_calibration")
+        if is_initial_calibration:
+            self.lid_handler._photo_creator.is_initial_calibration = (
+                is_initial_calibration
+            )
+            self.lid_handler._startStopCamera("initial_calibration")
```

Comment:
Can we create an enum/variable with the event/string? then we can use it on the lid_handler as well as here. Otherwise you really need to know what the method is doing and it could lead to problems soon if we forget to change the string in one of the places...